aclk/aclk_rx_msgs.c

```diff
@@ -256,3 +259,57 @@ int aclk_handle_cloud_message(char *payload)
 
     return 1;
 }
+
+void aclk_handle_new_cloud_msg(const char *message_type, const char *msg, size_t msg_len)
+{
+    // TODO do the look up table with hashes to optimize when there are more
+    // than few
+    if (!strcmp(message_type, "cmd")) {
+        aclk_handle_cloud_message(msg);
```

Comment:
Hi Timo! I get a warning here: `aclk/aclk_rx_msgs.c:268:35: warning: passing argument 1 of 'aclk_handle_cloud_message' discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]`